tool/dev.dart

```diff
@@ -27,7 +27,7 @@ main(List<String> args) async {
   config.test
     ..pubServe = true
     ..platforms = [
-      'dartium',
+      'chrome',
```

Comment:
Should we check the env and use dartium if on dart 1?

Reply:
We can, but the travis CI setup is much simpler if we run on chrome for dart1 and dart2

Reply:
🤷‍♂️ it's probably fast enough on chrome

Reply:
Nevermind!